<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddToConversationDiag</class>
 <widget class="QDialog" name="AddToConversationDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add to conversation</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,30">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="bottomMargin">
          <number>16</number>
         </property>
         <item alignment="Qt::AlignTop">
          <widget class="QLabel" name="CurrentlySelectedLabel">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Currently selected: &lt;span style=&quot; font-weight:600;&quot;&gt;None&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Add people to your conversation (up to 8):</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QListWidget" name="FriendsList"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
       <item>
        <widget class="QPushButton" name="CancelButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ConfirmButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
